templates/test/ci/prow-azl3/patches/controller-manager.yaml

```diff
@@ -0,0 +1,37 @@
+- op: add
+  path: /spec/kubeadmConfigSpec/files/0
+  value:
+    content: |
+      #!/bin/bash
+
+      set -o nounset
+      set -o pipefail
+      set -o errexit
+
+      # Install ca-certificates packages for Azure Linux
+      tdnf install -y ca-certificates ca-certificates-legacy
+      update-ca-trust
+      
+      # Follow Azure Linux 3 docs exactly - completely permissive for debugging
+      # Change default policy to ACCEPT (as recommended by AZL3 docs)
```

Comment:
> permissive for debugging

Did we need to change the `iptables` config for the cluster to work, or is this just for debugging purposes? That is, should we worry about tightening this up (now or in the future)?